subprojects/core/src/main/java/org/gradle/process/internal/JvmOptions.java

```diff
@@ -301,11 +306,15 @@ public void setEnableAssertions(boolean enabled) {
     }
 
     public boolean getDebug() {
-        return debug;
```

Comment:
❓ Should we take the time immediately and remove those two functions by refactoring the entire code base to remove its usage. I would suggest doing this as a follow-up PR but would try not to pile this up with the other debt that we need to clean.

Reply:
I'll create an issue/PR for this separately.